packages/wes_adapter/amazon_genomics/wes/adapters/CromwellWESAdapter.py

```diff
@@ -339,11 +339,7 @@ def _build_run_log_dict_(self, metadata, outputs):
                     "end_time": task.get("end"),
                     "stdout": task.get("stdout"),
                     "stderr": task.get("stderr"),
-                    "exit_code": (
-                        ""
-                        if task.get("returnCode") == None
-                        else str(task.get("returnCode"))
-                    ),
+                    "exit_code": task.get("returnCode")
```

Comment:
Do we expect `task.get("returnCode")` to be a number? If so you might be able to use `int(task.get("returnCode"))` to force it.  Also, what if `task.get("returnCode")` actually is `None`? If that happens you would need to return something that is a number otherwise it will fail when the CLI attempts to marshall the JSON.

How about:

```
exit_str = (
                    "" if task.get("returnCode") is None else task.get("returnCode")
                )
                try:
                    exit_code = int(exit_str)
                except (TypeError, ValueError):
                    self.logger.info(
                        f"exit code of '{int(task.get('returnCode'))}' is not an integer, will return 1 instead"
                    )
                    exit_code = 1

                log = {
                    "name": task_name + "|" + task.get("jobId", "XXXXX"),
                    "cmd": [task.get("commandLine")],
                    "start_time": task.get("start"),
                    "end_time": task.get("end"),
                    "stdout": task.get("stdout"),
                    "stderr": task.get("stderr"),
                    "exit_code": exit_code,
                }
                task_logs.append(log)
```

Reply:
We handle safely handle the "Null" use case on the Golang CLI side so we dont need to perform the check. You can take a look at that [here](https://cromwell.readthedocs.io/en/stable/api/RESTAPI/#definitions) (under CallMetadata - returnCode).